one_fm/public/js/doctype_js/job_applicant.js

```diff
@@ -739,18 +765,19 @@ var validate_min_age = function(frm) {
 };
 
 var create_interview = function(frm) {
-  frappe.route_options = {"job_applicant": frm.doc.name};
-	frappe.new_doc("Interview Result");
+	frappe.new_doc("Interview Result", {"job_applicant": frm.doc.name});
 };
-
+var check_doc_exist = function(frm, docname){
```

Comment:
I think you can remove this. Everything else looks good to me.

Reply:
Done